testsuite/Rakefile

```
@@ -129,4 +146,111 @@ namespace :utils do
     sh 'find . -size 0 -delete', verbose: false
     sh 'node index.js &> cucumber_reporter.log', verbose: false
   end
+
+end
+
+namespace :jenkins do
+
+  desc 'Generate rake file runstep'
+  task :generate_rake_run_step, [:extension, :client_name, :output_path, :yaml_path] do |t, args|
+    output_filepath = "#{args[:output_path]}/#{args[:client_name]}#{args[:extension]}.feature"
+    sh "echo '- #{output_filepath}' >> #{args[:yaml_path]}", verbose: false
+  end
+
+  desc 'Generate rake files for create bootstrap repositories'
+  task :generate_rake_files_create_bootstrap_repositories, [:instances] do |_t, args|
+    minions = args[:instances]
+    minions.each do |minion|
+      if minion.include?('ssh')
+        next
+      else
+        Rake::Task['jenkins:generate_rake_run_step'].invoke("_create_bootstrap_repository",
+                                                            File.basename(minion,".feature"),
+                                                            "features/build_validation/create_bootstrap_repositories",
```

```suggestion
        Rake::Task['jenkins:generate_rake_run_step'].invoke('_create_bootstrap_repository',
                                                            File.basename(minion,'.feature'),
                                                            'features/build_validation/create_bootstrap_repositories',
```
Single quotes can be used here since you do not use string interpolation. There are several occurrences for this in the other `Rake::Task`s below.